data_store/books_data.rb

```diff
@@ -0,0 +1,25 @@
+require 'json'
+
+module BooksData
+  def store_books(books)
+    books_array = []
+    path = './json_files/books.json'
+    return unless File.exist?(path)
+
+    books.each do |book|
+      books_array << { title: book.title, author: book.author }
+    end
+    File.write(path, JSON.generate(books_array))
+  end
+
+  def fetch_books
+    books_array = []
+    path = './json_files/books.json'
+    return books_array unless File.read(path) != '' && File.exist?(path)
```

Comment:
- [ ] Consider handling the case when the file is deleted by adding this line of code ` return [] unless File.exists?(path)` and add it for the rest of the files also.